src/Symfony/Component/Uid/Uuid.php

```diff
@@ -182,4 +182,36 @@ private static function format(string $uuid, string $version): string
 
         return substr_replace($uuid, '-', 23, 0);
     }
+
+    /**
+     * Transforms a binary string, a base-32 string or a base-58 string to a RFC4122 string.
+     *
+     * @param int-mask-of<Uuid::FORMAT_*> $format
+     *
+     * @return non-empty-string
+     */
+    private static function transformToRfc4122(string $uuid, int $format = self::FORMAT_RFC_4122): string
```

Comment:
```suggestion
    private static function toRfc4122(string $uuid, int $format): string
```

Reply:
A method with this name already exists in this class